docs/api/requirements.txt

```diff
@@ -1,5 +1,5 @@
 # pip install -r requirements.txt
 
-pydata-sphinx-theme==0.9.0
+pydata-sphinx-theme==0.10.1
```

Comment:
Careful with this update, so far there's been some weird breaking changes in the python dependency updates — I'd suggest only updating to the same version as solid-client or notifications

Reply:
Yeah, it has been taken care of by @kay-kim in https://github.com/inrupt/solid-client-access-grants-js/pull/320, and removed from the current PR.